pkg/util/proxyconfig/no_proxy_test.go

```diff
@@ -176,7 +197,7 @@ func TestMergeUserSystemNoProxy(t *testing.T) {
 			args: args{
 				proxy:   proxyConfigWithNoProxy("172.30.0.1."),
 				infra:   infraConfig(configv1.AWSPlatformType, "^&", "us-west-2"),
-				network: netConfig("10.128.0.0/14", ""),
+				network: netConfig("10.128.0.0/14", []string{""}),
```

Comment:
Any reason to explicitly test with `[]string{""}` instead of `nil`?

Reply:
No reason, I will change it